### PR TITLE
Cu-24nu2ed mines handler unit tests

### PR DESCRIPTION
**Pull request by rkamysz**
PR includes:

- refactored code to make it testable
- unit tests for the mines handler
- mines fixtures
- mocked fastify instance

and some lint changes in other files

### CODE REVIEW

**Comment by dskvr** [comment]
Task linked: [CU-24nu2ed Write unit tests for mines handler](https://app.clickup.com/t/24nu2ed)

File: src/api-handlers/__tests__/fixtures/mines.fixture.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  block_timestamp: {
+    date: '2022-02-04T14:34:53Z',
+  },
+  global_sequence: 37593403954,
+  tx_id: '91f0f074f841cafba129c28f65e551f022f7a2b5ba2fdc3463291fc7aee29ce1',
+};
+
+export const sampleMine: Mine = {
+  id: '61fd3a3bcc6d81005d135799',
+  miner: '4uzia.wam',
```
Comment (montera82) [inline src/api-handlers/__tests__/fixtures/mines.fixture.ts:31]: do you know if we should obfusicate out all these out...they are like PII to me...although to be honest they are also on the public blockchain ...up to you

**Review by montera82 — APPROVED** [review]
glad the code is finally getting a bath!
LGTM

**Review by montera82 — APPROVED** [review]

File: src/api-handlers/__tests__/fixtures/mines.fixture.ts (hunk 1 above)
Comment (rkamysz) [reply src/api-handlers/__tests__/fixtures/mines.fixture.ts]: changed and actually that should be one of the points in our how to write tests to use only fake data for unit tests or our internal for e2e tests, not clients/users etc

**Pull request merged**